docs/source/developer_documentation.rst

```diff
@@ -0,0 +1,128 @@
+Developer Documentation
+=======================
+
+Github Workflow
+---------------
+
+-  You should request reviews for pull requests from ``multisignal-epi-inference-devs``, unless you have a good reason to request reviews from a smaller group.
+-  Reviews from all of ``multisignal-epi-inference-devs`` are  encouraged, but an approving review from @dylanhmorris, @damonbayer, or @gvegayon is required before a pull request can be merged to ``main``.
+-  If your pull request has not received a review at the time of the next standup, use standup to find a reviewer.
```

Comment:
Should we have a sentence here for CFA-external developers, who are unlikely to know what "standup" means or how to use it? Perhaps this could be something along the lines of "External developers should expect their PRs to be reviewed in 2 days time."

Reply:
Good idea. Addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/224/commits/5e1cd68ad691f2f4be7c5fd1812dc6295c8abf17